src/spec-configuration/containerFeaturesConfiguration.ts

```diff
@@ -47,6 +47,7 @@ export interface Feature {
 	included: boolean; // set programmatically
 	customizations?: VSCodeCustomizations;
 	installsAfter?: string[];
+	legacyIds?: string[];
```

Comment:
Please be sure to add this to the schema (https://github.com/devcontainers/spec/blob/main/schemas/devContainerFeature.schema.json) when you're ready to!

Reply:
Yep, I'll start with the spec PR in parallel. Thanks.

Reply:
https://github.com/devcontainers/spec/tree/samruddhikhandale/features-add-legacyIds